test/unit/User.test.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-env mocha */
+import React from 'react';
+import sinon from 'sinon';
+import { expect } from 'chai';
+// Import the component that is going to be tested
+import User from './../../src/server/ApiRoutes/User.js';
+
+let mockTokenResponse = {};
+const mockRes = { redirect: () => {}, };
+
+const renderMockReq = (mockTokenResponse) => {
+  return {
+    get: (n) => { return n; },
+    protocol: 'http',
+    originalUrl: '/hold/request/b11995345-i14211097',
+    tokenResponse: mockTokenResponse,
+  };
+};
+
+describe('If requireUser does not receive valid value from "req.tokenResponse"', () => {
+  let requireUser;
+
+  before(() => {
+    requireUser = sinon.spy(User, 'requireUser');
+  });
+
+  after(() => {
+    mockTokenResponse = {
```

Comment:
Yeah you are right. I did this to avoid repeated codes. And you also are right if we think about the independency of each test. I will update them.